pkg/grafana/datasources_default.go

```diff
@@ -0,0 +1,71 @@
+package grafana
+
+import "fmt"
+
+const (
+	// datasourceUIDPrefix is the prefix for all datasources managed by the operator
```

Comment:
Make sure to document that the `gs-` UID prefix is reserved for GiantSwarm and will be deleted.